migrations/20160406175457_add_meta_fields_to_entities.js

```diff
@@ -0,0 +1,50 @@
+var tables = [
+    'interventions',
+    'locations',
+    'organisations',
+    'persons',
+    'problems',
+    'records',
+    'sources',
+    'trials',
+]
+
+exports.up = (knex) => {
+
+  var operations = []
+  tables.forEach((table) => {
+      var addTimestamps = knex.schema.table(table, (table) => {
+          table.timestamp('created_at').nullable();
+          table.timestamp('updated_at').nullable();
```

Comment:
Instead of setting those, use `table.timestamps();`. It creates the `created_at` and `updated_at` columns as `nonNullable` and sets them to the current time by default. There might be some problems given that we're not starting from a clean database, but I don't think so.